tests/test_mgp_unit_ff.py

```diff
@@ -0,0 +1,221 @@
+import numpy as np
```

Comment:
mgp_en.py and mgp.py are quite duplicated. I'll consider merging them, and set up options for "energy map" or "force map"